From ef59c3c0bdc0a2e91915274963e9cfbfb9ec459f Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Fri, 14 Jun 2019 17:00:45 +0000
Subject: [PATCH] MANTA-4347 get boray and electric-boray building in Jenkins
 Reviewed by: Angela Fong <angela.fong@joyent.com> Approved by: Angela Fong
 <angela.fong@joyent.com>

---
 Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Makefile b/Makefile
index 73a76a6..9b430ce 100644
--- a/Makefile
+++ b/Makefile
@@ -30,7 +30,6 @@ ROOT :=			$(shell pwd)
 RELSTAGEDIR :=		/tmp/$(NAME)-$(STAMP)
 
 BASE_IMAGE_UUID = cbf116a0-43a5-447c-ad8c-8fa57787351c
-BUILD_PLATFORM  = 20180605T172856Z
 BUILDIMAGE_NAME = manta-boray
 BUILDIMAGE_DESC	= Manta boray proxy
 AGENTS		= amon config registrar
-- 
2.21.0

